docs/content/how-to/baremetal-none.md

```diff
@@ -0,0 +1,427 @@
+# HyperShift + Bare Metal workers using 'none' provider
```

Comment:
This renders as the title, see https://deploy-preview-1079--hypershift-docs.netlify.app/how-to/agent-baremetal/

Since this is Hypershift docs can we may be drop "Hypershift" and leave it as "Bare Metal workers using 'none' provider"

Reply:
Fixed! Thanks for your inputs.